### PR TITLE
docs: Update Lamp to include Android specific requirements

### PR DESCRIPTION
**Pull request by morning4coffe-dev**
GitHub Issue (If applicable): part of #10614 

## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/articles/features/flashlight.md [hunk 1]
```diff
@@ -26,6 +26,17 @@ uid: Uno.Features.Flashlight
 * On **iOS**, in case the device supports the torch, `BrightnessLevel` is fully supported. In case the device has only flash, any non-zero `BrightnessLevel` will result in the full brightness of the flashlight.
 * On **Android**, flashlight brightness cannot be controlled, hence any non-zero `BrightnessLevel` results in the full brightness of the flashlight.
 
+## Platform-specific
```
Comment (Youssef1313) [inline doc/articles/features/flashlight.md:29]:
```suggestion
## Platform-specific requirements
```

File: doc/articles/features/flashlight.md [hunk 2]
```diff
@@ -26,6 +26,17 @@ uid: Uno.Features.Flashlight
...
+
+### Android
+
+For Android there are two permissions you must configure before using this API in your project:
```
Comment (Youssef1313) [inline doc/articles/features/flashlight.md:33]:
```suggestion
For Android, there are two permissions you must configure before using this API in your project. To do that, add the following to `AndroidManifest.xml`:
```

Comment (morning4coffe-dev) [reply doc/articles/features/flashlight.md:33]: Thanks!

**Review by jeromelaban — APPROVED** [review]

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.1

**Pull request merged**